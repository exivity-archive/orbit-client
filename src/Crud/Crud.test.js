/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'

test('renders div', () => {
  const alert = renderer.create(<div />)
  expect(alert.toJSON()).toMatchSnapshot()
})
