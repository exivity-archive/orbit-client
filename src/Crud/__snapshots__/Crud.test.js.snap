// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders div 1`] = `<div />`;
